packages/dx-react-grid-material-ui/src/templates/layout.test.jsx

```diff
@@ -1,66 +1,97 @@
 import React from 'react';
-import { createMount, getClasses } from 'material-ui/test-utils';
-import { setupConsole } from '@devexpress/dx-testing';
-import { Header, Footer } from './layout';
+import { shallow } from 'enzyme';
+import { getClasses, createShallow } from 'material-ui/test-utils';
```

Comment:
Is this necessary to use `shallow` and `createShallow` at the same time?

Reply:
@SergeyAlexeev Yes, because enzume [docs](http://airbnb.io/enzyme/docs/api/ShallowWrapper/shallow.html) have this note: "can only be called on wrapper of a single node"